ironic/drivers/modules/drac/management.py

```diff
@@ -84,6 +89,24 @@
 # Clean steps constant
 _CLEAR_JOBS_CLEAN_STEPS = ['clear_job_queue', 'known_good_state']
 
+# Jobs base url
+_JOBS_BASE_URL = '/redfish/v1/Managers/iDRAC.Embedded.1/Jobs'
+
+# iDRAC reset base url
+_RESET_URL = '/redfish/v1/Managers/iDRAC.Embedded.1/Actions/Manager.Reset/'
+
+# iDRAC remote service api status constant
+_REMOTE_SERVICE_API_STATUS = (
+    '/redfish/v1/Dell/Managers/iDRAC.Embedded.1/'
+    'DellLCService/Actions/DellLCService.GetRemoteServicesAPIStatus')
+
+_HEADERS = {'content-type': 'application/json'}
+
+# idrac remote service status retries constant
+_READY_RETRIES = 24
```

Comment:
We should make the above constant have the same value as DEFAULT_IDRAC_IS_READY_RETRIES in python-dracclient:
https://github.com/openstack/python-dracclient/blob/master/dracclient/constants.py#L15

Reply:
Addressed. Used this constant values same DEFAULT_IDRAC_IS_READY_RETRIES.